helm-chart/splunk-connect-for-kubernetes/values.yaml

```diff
@@ -1,29 +1,898 @@
-# Splunk Connect for Kubernetes is a umbraller chart which itself does not contain any template,
-# so there is no configurable parameters for itself.
+# Splunk Connect for Kubernetes is a umbraller chart for three charts
+# * splunk-kubernetes-logging
+# * splunk-kubernetes-objects
+# * splunk-kubernetes-metrics
 
 # Use global configurations for shared configurations between sub-charts.
 # Supported global configurations:
-#
-# global:
-#   logLevel:
-#   splunk:
-#     # HEC related configurations
-#     hec:
-#   kubernetes:
-#     clusterName: "cluster_name"
-# For other configurations for sub-charts, please check their values.yaml for details.
-
-## Enabling logging will install the `splunk-kubernetes-logging` chart to a kubernetes
+# Values defined here are the default values.
+global:
+  logLevel: info
+  splunk:
+    hec:
+      protocol: https
+      port: 8088
+      insecureSSL: false
+  kubernetes:
+    clusterName: "cluster_name"
+
+
+## Enabling splunk-kubernetes-logging will install the `splunk-kubernetes-logging` chart to a kubernetes
 ## cluster to collect logs generated in the cluster to a Splunk indexer/indexer cluster.
-logging:
+splunk-kubernetes-logging:
   enabled: true
+  # logLevel is to set log level of the Splunk log collector. Avaiable values are:
```

Comment:
why are we repeating all the configs from the individual charts here?
This will be really hard to maintain.

Reply:
it is a single place for users to control SCK. Regarding maintenance, maybe we can write a script that will update main repo values.yaml file from sub charts?

Reply:
When all things you can control for SCK is not in one place, users would have to move back and forth.